### DISCUSSION TITLE
ED113TC1 compatability

### DISCUSSION THREAD

**Discussion by SpandexWizard** [vroland/epdiy#296] [General]
so i'm pretty sure the epdiy is NOT compatible with the display i want to use. per the datasheet (https://www.data-modul.com/sites/default/files/products/VB3300-LCA-specification-12053273.pdf) the positive gate supply is supposed to be 25v. as far as i can tell, the epdiy only supplies 22v, which is outside the tolerances of the datasheet. is there some way to address this? i presume that the tps651851rs would have to be replaced with something that CAN produce 25v, as a start... is there such a chip? i know that it would mean reworking the whole board from the ground up probably, but my usecase probably demands that anyway.

**Comment by martinberlin** [comment]
This is like this with many other panels too but still the FET gates that is what really this voltages are for (as +15 and -15V are used by all Eink to drive the pixels) works with the standard voltage settings.
Read the TPS65185 data sheet as is possible to change this with some resistance values update.

I would build a v7 or purchase one and try it as is.
I’m offering a custom v7 that has a 13 inch connector in Tindie.

**Reply by SpandexWizard** [reply]
are there examples of other displays that have higher positive voltages than 22v that do work? i'm not really in a position to buy hardware just to see if it will work. especially when the numbers say it wont?

**Reply by schuhumi** [reply]
You slightly misread the model number Martin, it's 11.3 not 13.3 inches. Also it sports a different comnector with different pincount (50 instead of 39)

**Reply by martinberlin** [reply]
Ah ok but about the voltages so far most worked out correctly with the voltages per default even though on the data sheet had another values. 
Another very different topic is if any of the open source waveforms will work with the display. Most probably will require manual modifications

**Reply by vroland** [reply]
@SpandexWizard  Actually, according to the TPS65185 data sheet you can get the voltage of the `CP1 (VDDH) CHARGE PUMP` up to 28V by modding a resistor value. So if you have a board you can try to replace the 47.5kOhm resistor for a 41.6kOhm resistor. If that works, you can give driving the actual display a shot. Just make sure you don't accidentally destroy other displays with your modded board ;)

**Reply by SpandexWizard** [reply]
interesting!

**Comment by martinberlin** [comment]
Give a read to the issues and discussions here. 
Many other displays have slightly different voltage settings and so far all that I’ve tried work with this standard voltages

**Reply by martinberlin** [reply]
Just a note @SpandexWizard 

> i'm not really in a position to buy hardware just to see if it will work

If you want to try still not supported displays you need to take the risk and go for it.
Me as collaborator of epdiy or Valentin himself we take this risks from time to time and try new displays but we cannot possibly try it for every display.

**Reply by SpandexWizard** [reply]
i definitely don't expect you guys to try my display for me. XP  but hearing that others have worked outside of their spec is helpful

**Reply by martinberlin** [reply]
I've heard sometimes that displays with slightly higher FET gate voltages worked correctly with epdiy standard settings.
Sadly this was in https://epdiy.slack.com non-paid version and messages are deleted after 90 days.
I would try first with standard voltages and then only if that does not work, as VRoland mentioned up there change the 47.5kOhm resistor for a 41.6kOhm resistor. But make sure not to switch displays because maybe another one with lower voltage might be fried

**Comment by martinberlin** [comment]
Related to this discussion #298 please give it a read also

**Comment by zephray** [comment]
I can confirm that ED113TC1 works with 22V gate voltage (tested myself). But keep in mind this is out of spec, your mileage may vary.
